### PR TITLE
Allowed images to be sent from s3-uploader

### PR DESCRIPTION
**Pull request by currysoup**
Images can now be added from S3 uploader:

A valid image:
imageupload

A "valid" URL which does no have an image at the end:
imageupload_notreal

### CODE REVIEW

**Comment by na7hangood** [comment]
Nice. I want that Pikachu contributor tag too

**Comment by currysoup** [comment]
New error message:

imageupload-newmessage

**Comment by currysoup** [comment]
Changing delete icon to a bin (was previously a cross): 

imageupload-delete

Bye bye Pikachu

**Comment by clloyd** [comment]
:+1: @paperboyo

**Comment by paperboyo** [comment]
Great! Thanks, all!
> I want that Pikachu contributor tag too

TagManager is so easy to use and fast, that I just made one for you:

pikachu

File: public/util/validateImage.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const URL_REGEX = /^https:\/\/static.guim.co.uk\/sys-images\/Guardian\/.*\.(png|jpg)$/;
+const URL_REGEX = /^https:\/\/(static.guim.co.uk\/sys-images\/Guardian|uploads.guim.co.uk)\/.*\.(png|jpg)$/;
```
Comment (akash1810) [inline public/util/validateImage.js]: Note s3-uploader maintains the original filename, so you could get `.jpeg` files.

**Pull request merged**